plugins/file/index.js

```diff
@@ -75,7 +75,10 @@ module.exports = {
           return Boom.badData("Failed to read file");
         }
 
-        const contentType = file.hapi.headers["content-type"];
+        let contentType = file.hapi.headers["content-type"];
```

Comment:
can you add a comment here pointing to the whatwg spec?

Reply:
Sure. Added in https://github.com/nzzdev/Q-server/pull/135/commits/4cc18b8434834a9a929d787209213fc192a6790d